src/test/parallel/codecept.conf.js

```diff
@@ -32,26 +46,26 @@ exports.config = {
                         "browserstack.debug": "true",
                         'browserstack.networkLogs': 'true',
                     },
-                    //For W3C-based scripts, use the following capabilties:
-                    /*
-                    desiredCapabilities: {
-                     "bstack:options" : {
-                       "os": "OS X",
-                       "osVersion": "Catalina",
-                       "projectName": "Codecept + WebdriverIO",
-                       "buildName": "Parallel_Execution",
-                       "sessionName": "Parallel Test Safari",
-                       "debug" : "true",
-                       "networkLogs" : "true",
-                     },
-                     "browserVersion": "latest",
-                    },
                     */
                 },
 
                 {
                     browser: "Firefox",
+                    //For W3C-based scripts, use the following capabilties:
+                    desiredCapabilities: {
```

Comment:
capabilities?

Reply:
done. although it was working with that as well